npm-shrinkwrap.json

```diff
@@ -731,7 +731,7 @@
     },
```

Comment:
Unless we've added new packages, we should not commit changes to npm-shrinkwrap.json